packages/mongo-nav/src/project-nav/ProjectNav.tsx

```diff
@@ -430,7 +435,9 @@ export default function ProjectNav({
                 className={iconButtonMargin}
                 size="large"
                 disabled={!current}
-                active={activeNav === ActiveNavElement.ProjectNavInvite}
+                active={
+                  activeNav === ActiveNavElement.ProjectNavInvite && !!current
```

Comment:
May want to cache the `!!current` variable.

Reply:
Done 👍